src/api.cpp

```diff
@@ -1466,11 +1476,26 @@ cvk_create_command_queue(cl_context context, cl_device_id device,
         return nullptr;
     }
 
+    cl_int err = CL_SUCCESS;
+    std::string oooe_env_var = "CLVK_IGNORE_OUT_OF_ORDER_EXECUTION";
```

Comment:
Please use clvk configuration (`<clvk>/src/config.def`). This should be a boolean there. It will allow to use the env var as well as the configuration file.